padl/dumptools/inspector.py

```diff
@@ -323,6 +218,35 @@ def caller_frame() -> types.FrameType:
     return outer_caller_frameinfo(calling_module_name).frame
 
 
+def instructions_are_the_same(instr, target_instr, frame=None):
```

Comment:
Would be great to have docstrings for the parameters.

Reply:
:+1: